lib/jsdom/utils.js

```diff
@@ -145,6 +145,33 @@ exports.treeOrderSorter = function (a, b) {
   return 0;
 };
 
+/**
+ * @param {ArrayBuffer} buffer
+ * @returns {boolean}
+ */
+
+exports.isDetachedBuffer = function (buffer) {
+  return buffer.byteLength === 0;
+};
```

Comment:
https://github.com/jsdom/webidl-conversions/blob/7b8f708ffd7476fb60d18cef75bbdf4e60831191/lib/index.js#L329-L337 is how we've done this previously. Unfortunately that isn't exported, but oh well.

Reply:
Should I export the function you linked or should we keep the one I made?

Reply:
Let's duplicate that function here in utils.js. It'd be a bit weird for the webidl-conversions project to export it, and, it's not worth a whole release cycle for that other project anyway.